src/app/js/home/Home.js

```diff
@@ -1,3 +1,6 @@
 import React, { PropTypes } from 'react';
+import Upload from './Upload';
```

Comment:
If #7 is merged first, you'll be able to put this Upload component on the Admin component